src/wasm.rs

```diff
@@ -259,7 +268,65 @@ pub fn genWitness(
         .forward(&mut input, None, None)
         .map_err(|e| JsError::new(&format!("{}", e)))?;
 
-    serde_json::to_vec(&witness).map_err(|e| JsError::new(&format!("Failed to serialize witness: {}", e)))
+    serde_json::to_vec(&witness)
+        .map_err(|e| JsError::new(&format!("Failed to serialize witness: {}", e)))
+}
+
+/// Generate proving key in browser
+#[wasm_bindgen]
+#[allow(non_snake_case)]
+pub fn genPk(
```

Comment:
hey could we flip this around ? so if you look at `create_keys_wasm` it first creates the vk and then the pk as such 

```rust

    // Initialize the proving key
    let vk = keygen_vk(params, &empty_circuit)?;
    let pk = keygen_pk(params, vk, &empty_circuit)?;

```

The wasm function should be such that `genVK` takes in compiled model and srs and generates the  VK, and `genPK` then takes in the VK and params to generate the PK . 

This will be particularly useful for the stats project as it means they only need to generate the VK and not the PK + VK to get the VK (half the work)